lib/common/buffer-utils.ts

```diff
@@ -19,16 +14,16 @@ const typedArrays = {
   Float64Array
 }
 
-function getType (value) {
+function getType (value: BufferSource) {
```

Comment:
hm... actually this should also include `Buffer` which `BufferSource` doesn't.

maybe `type BufferLike = ArrayBuffer | ArrayBufferView | Buffer`?

Reply:
`Buffer` is now actually `Uint8Array`, but it's treated specially to keep it transferred as `Buffer`